tests/e2e/testRunner.ts

```diff
@@ -191,6 +191,7 @@ const runTests = async () => {
                     // maximum number of allowed exceptions, we should stop the test run.
                     throw e;
                 }
+                // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
                 Logger.warn(`There was an error running the test. Continuing the test run. Error: ${e}`);
```

Comment:
Will it work? But I'm okay to leave it as is if we are afraid of regression for some reason
```suggestion
                Logger.warn(`There was an error running the test. Continuing the test run. Error: ${e.message}`);
```

Reply:
Exactly, I'm a bit afraid of regressions 😅